subjects/prime_checker/README.md

```diff
@@ -0,0 +1,43 @@
+## prime_checker
+
+### Instructions
+
+Create a **function** `prime_checker` that takes an `u32` and check if it is a prime number.
+
+The result will be `None` if the argument is less or equal one otherwise, it will return the result with the prime number inside if the number is prime, or it will return an `enum` `PrimeErr` if the `u32` is not prime.
+
+The `enum` `PrimeErr` will be `Even` if the number is even or `Divider(u32)` where the `u32` is the smaller divider of the number if it is not prime.
+
+### Expected Function and structure
+
+```rust
+#[derive(PartialEq, Eq, Debug)]
+pub enum PrimeErr {
+    Even,
+    Divider(u32),
+}
+
+pub fn prime_checker(nb: u32) -> Option<Result<u32, PrimeErr>> {}
```

Comment:
It would be nice if by giving more details in the previous part we could hide the return type from this prototype, the student will then be required to know and understand the basic behavior of `Option` and `Result` which are the focus of this exercise.